src/test/java/scripts/ShellScriptTest.java

```diff
@@ -73,7 +73,7 @@ public File getScriptFile() {
 
     @Test
     void testInnerSpaces() throws IOException, InterruptedException {
-        final boolean success = testEscapingSpaces("--detect.project.name=Synopsys Detect");
+        final boolean success = testEscapingSpaces("--detect.project.name=Blackduck Detect");
```

Comment:
Just Detect

Reply:
I decided to keep it this way as this is a test and it's actually testing specifically that the space between two words doesn't break anything.